src/styles/_catalog.scss

```
@@ -1,22 +1,34 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
-  margin: 0 auto;
-
+  display: grid;
+  grid-template-columns: repeat(4, 200px);
+  grid-template-rows: repeat(auto-fill);
+  grid-auto-rows: 300px;
+  justify-content: center;
+  gap: 48px;
+  margin-bottom: 48px;
   &__card {
     display: flex;
     justify-content: center;
     align-items: center;
-    width: 200px;
-    height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
-    margin-bottom: 48px;
   }
+}
+
+@media (max-width: 1024px) {
+  .catalog {
+    grid-template-columns: repeat(3, 200px);
```

```suggestion
    grid-template-columns: repeat(3, 200px);
```
move 200px to the variable